cartographer/sensor/compressed_point_cloud.h

```diff
@@ -50,13 +50,18 @@ class CompressedPointCloud {
 
   proto::CompressedPointCloud ToProto() const;
 
+  CompressedPointCloud(const proto::CompressedPointCloud& proto);
```

Comment:
Style: Constructors come first, i.e. move write below the `explicit CompressedPointCloud(`. Also: Make `explicit` since it takes a single argument.